### ISSUE TITLE
Ignoring CUSTOM_HTTPS_KEY & invalid number of arguments in "ssl_certificate_key" directive 

### ISSUE THREAD

**Issue by souchprod** [bunkerity/bunkerized-nginx#140]
Hello, 

bunkerized-nginx is not working anymore in my case since the latest update (not sure if very latest, but change occurred in the last 2 weeks for sure).

Log
```
  [*] Starting bunkerized-nginx ...                  
  [*] Configuring bunkerized-nginx ...               
  Ignoring LOGROTATE_MAXAGE=30                       
  Ignoring CUSTOM_HTTPS_KEY=/certs/........../.........._key.pem
  Ignoring USE_LIMIT_REQ=80r/s                       
  Ignoring USE_PHP=no                                
  Ignoring USE_FAIL2BAN=no                           
  [*] Generation done !                              
  [*] Generated self-signed certificate for default server
  [*] Downloading bad user-agent list (in background) ... 
  [*] Downloading bad referrer list (in background) ...
  [*] Downloading tor exit nodes list (in background) ...
  [*] Downloading proxies list (in background) ...   
  [*] Downloading abusers list (in background) ...   
  [*] Running nginx ...                              
  [2021-06-09 10:05:19] [BLACKLIST] user-agent list updated (1710 entries)
  [2021-06-09 10:05:19] [BLACKLIST] referrers list updated (7059 entries)
  nginx: [emerg] invalid number of arguments in "ssl_certificate_key" directive in /etc/nginx/https.conf:8
  [*] bunkerized-nginx stopped    
```

Docker stack content:

```
environment:
      USE_CUSTOM_HTTPS: "yes"
      CUSTOM_HTTPS_CERT: "/certs/............/.............crt"
      CUSTOM_HTTPS_KEY: "/certs/............/............_key.pem"
      AUTO_LETS_ENCRYPT: "no"
      REDIRECT_HTTP_TO_HTTPS: "yes"
```

No idea why CUSTOM_HTTPS_KEY is ignored here ?

[EDIT] Rollbacking to 1.2.5 = no problem, bug or breaking change in version 1.2.6.

**Comment by bunkerity** [comment]
Hello @souchprod,

I may be related to the [regex](https://github.com/bunkerity/bunkerized-nginx/blob/master/settings.json#L591) of the `CUSTOM_HTTPS_KEY` value which is the following : 
```regex
^[A-Za-z\.0-9\/\-]*$
```

I see an underscore in your path which isn't allowed by the regex. Do you have some time to rename your file, edit the `CUSTOM_HTTPS_KEY` and try again ? If that's the problem, I will edit it to add the underscore. Thanks for reporting this.

**Comment by souchprod** [comment]
Well done, indeed removing the underscore resolve the problem 👍 
Thanks to include the underscore, it may help others.

Thx for your work

**Comment by bunkerity** [comment]
Keeping this open as a reminder.

**Comment by bunkerity** [comment]
Fixed in v1.2.7 : added the underscore and also more info in the log when a variable is ignored. Thanks again !
